lib/dotcom_web/templates/schedule/_stop_list_expand_link.html.eex

```diff
@@ -8,16 +8,27 @@
     |> DotcomWeb.PartialView.StopBubbles.render_stop_bubbles(true, stop_branch)
   %>
   <div class="route-branch-stop-info expand">
+    <div class="d-flex">
+      <div>
+        <% duration_minutes = DotcomWeb.TripPlanView.display_seconds_as_minutes(@itinerary_row.duration) %>
+        <%= DotcomWeb.TripPlanView.icon_for_route(@route) %>
+      </div>
+      <div class="ps-8">
+        <span class="u-bold pe-8">Ride the <%= @branch_display %> </span> (<%= intermediate_stop_count %> stops, <%= duration_minutes %> minutes)
```

Comment:
Was removing the `Inflex.inflect/2` bit intentional? That's used to dynamically pluralize the word (e.g. stop or minute) based on the number of things being counted, it seems like we'd want to keep that there?

Reply:
Thats an oversite